packages/web/src/components/createCell.tsx

```diff
@@ -28,79 +24,184 @@ export type CellProps<
     (GQLVariables extends { [key: string]: never } ? unknown : GQLVariables)
 >
 
-export type DataObject = { [key: string]: unknown }
+type ScrubbedQueryOperationResult = Omit<
+  QueryOperationResult,
+  'loading' | 'error' | 'data'
+> & {
+  updating: boolean
+}
```

Comment:
Tried reduce some of the duplication in the typing but didn't know what to name it

Reply:
yeah the name is difficult, but to me it sounds like this is the QueryOperationResult without any of the properties we add... but for some reason we also include updating 🤣

Reply:
I reverted the change for now—if I have a better idea I'll add something but I don't yet

Reply:
Yeah makes sense 👍. I know its "duplicated" but seems more logical this way